examples/internal/helloworld/BUILD.bazel

```diff
@@ -1,40 +1,29 @@
-load("@rules_proto//proto:defs.bzl", "proto_library")
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
-load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
-# gazelle:exclude helloworld.pb.gw.go
-# gazelle:exclude helloworld_grpc.pb.go
 # gazelle:go_grpc_compilers //:go_apiv2, //:go_grpc, //protoc-gen-grpc-gateway:go_gen_grpc_gateway
 
-proto_library(
```

Comment:
Won't users in not-go sill need to depend on the `proto_library`?

Reply:
It looks like having both proto_library and and go_library would cause some build warning with Gazelle command. If there is a way to keep both, I agree that would be ideal - but I'm not sure how to achieve that 😥